test/homepageFunc.js

```diff
@@ -0,0 +1,21 @@
+import Data from './__mock__.js';
+
+export default function moviesCounter(n) {
+  const data = new Data();
+  data.setData(n);
+  const movies = data.getData();
+
+  if (movies) {
+    if (movies.length <= 100) {
+      document.querySelector('#episode-counter').innerHTML = `Episode(${movies.length})`;
+
+      return movies;
+    }
+
+    document.querySelector('#homepage-ul').innerHTML = 'Too many movies !!';
+  } else {
+    document.querySelector('#homepage-ul').innerHTML = 'fetching failed !';
+  }
+
+  return null;
+}
```

Comment:
- [ ] The same happens here, please avoid creating new functions to test. The main idea of testing is to give you confidence that your program works as expected. However, if you're using different functions in your program and your tests, this confidence is null. Please consider testing the original functions. 🤓 🔧 


💡 _I suggest that you refactor your counters to count the DOM elements instead of the length of the array returned by the API, this would make testing easier since you don't have to handle async logic_

Reply:
Hello @anagudelogu ,
The same for this case, the function I need to make test is "updateComment()", right?

Reply:
Hello @anagudelogu ,
By the way, I just mocked the data to have the input.I did not change the function.

Reply:
Thank you @anagudelogu ,
I got the idea. 
I modified all item-counter modules to calculate all the items in the DOM.